source/user_guide/case_auditing.rst

```diff
@@ -0,0 +1,138 @@
+
+
+Case Importance
+---------------
+
+As opposed to influential and similar cases which examines the influence of cases on a single case or prediction, case importance examines how important a case is in regards to the overall predictions on a group of cases. Case importance share the same underlying methodology with  :doc:`Feature Importance <feature_importance>`.
+Unlike feature contributions, case contributions are calculated just locally. Conceptually, local metrics use either a specific subset of the cases that are trained into the Trainee or a set of new cases.
+
+Robust vs Non-Robust
+^^^^^^^^^^^^^^^^^^^^
+In order to calculate case importance, Howso Engine measures the impact on the prediction by comparing predictions with and without the case. The case set without the case of interest may include either all of the other cases, or a combination of cases that
+may include any number of other cases. Non-robust calculations use a leave-one-out approach to calculate these metrics, thus the metrics reflect the results when all cases expect the case of interest is used. Robust case contributions instead sample from
+the power set set of all combinations of cases without the case of interest. Robust metrics are recommended as they encompass a greater variety of case sets, and they include a calculation performance boost as the number of cases increases.
+
+
+To get the case importance metrics, :py:meth:`Trainee.react`, is first called on a trained and analyzed Trainee. In this method, the desired metrics, ``case_contributions`` and ``case_mda``, can be selected as inputs to the ``details`` parameters as key value pairs from a dictionary. These parameters are named individually
+and setting them to ``True`` will calculate the desired metrics. To calculate the robust versions, ``robust_computation`` is set to True.
+
+.. code-block:: python
+
+    details = {
+        'robust_computation':True,
+        'case_contributions':True,
+        'case_mda':True,
+    }
+
+    results = t.react(
+        df,
+        context_features=context_features,
+        action_features=action_features,
+        details=details
+    )
+
+These metrics can be retrieved the same way as the method to retrieve the most similar cases. Whether these metrics are robust or non-robust is determined when the metrics
+are calculated in :py:meth:`Trainee.react` from the previous step.
+
+.. code-block:: python
+
+    robust_case_contributions = results['explanation']['case_contributions']
+    robust_case_contributions = results['explanation']['case_mda']
+
+
+.. warning::
+
+    Contributions and MDA are also metrics for features and not just cases, so please be aware when reading other guides that may use those terms.
+
+
+Example Use-Cases
+^^^^^^^^^^^^^^^^^
+In addition to the examples above, here are a few example use-cases for for similar cases and case importance.
+
+- Prediction Auditability and Tracing
```

Comment:
commas!!

Reply:
Prediction aditability was supposed to be one phrase, I'll get rid of prediction though